up42/tools.py

```diff
@@ -245,6 +245,9 @@ def plot_results(
             if self.results is None:
                 raise ValueError("You first need to download the results!")
             filepaths = self.results
+            # Flatten jobcollection results dictionary.
+            if isinstance(filepaths, dict):
+                filepaths = [item for sublist in list(filepaths.values()) for item in sublist]
```

Comment:
Is a bit hard for me to understand what is happening is this list compehension...